.github/workflows/tag-version.yml

```diff
@@ -0,0 +1,46 @@
+name: Tag Version
+
+on:
+  push:
+    branches:
+    - main
+
+jobs:
+  tag-version:
+    runs-on: ubuntu-latest
+    if: "contains(github.event.head_commit.message, ':bookmark: Version ')"
+
+    steps:
+    - uses: actions/checkout@v4
+      with:
+        fetch-depth: '0'
+
+    - name: Tag version
+      run: |
+        msg_start=':bookmark: Version '
```

Comment:
Any way to trigger an error & fail the workflow if another message that follows `:bookmark:` isn't `Version `?

Reply:
Done